wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/wss/util/WSSServletContextFacade.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022 Eclipse Foundation and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
```

Comment:
This license is WRONG - must be EDL as is the rest of the code base. I'll change it.

Reply:
btw: nightly build job is saying that in https://ci.eclipse.org/metro/job/wsit-master-build/124/console (it's linked from the project home page) and notification was sent to metro-dev list

Reply:
Aha, I had a discussion with Arjan yesterday, he noticed somewhere that the copyright plugin takes just the first line to parse the year, so the latest should be first. Until now I always added just the second line. 
Sorry for the license, I forgot it is different from Glassfish.

Reply:
it can check multiple things - for the year you are correct, it takes only the first line (unless copyright.ignoreyear is true), but it also checks the license as such either against arbitrary template (copyright.template or copyright.alternateTemplate (or sth similar) properties) or against some predefined ones packaged within the copyright plugin